lib/resty/acme/autossl.lua

```diff
@@ -78,7 +78,7 @@ local function get_certkey(opts)
   local typ = opts.type
   local domain = opts.domain
   local data, _ --[[stale]], _ --[[flags]] = certs_cache[typ]:get(domain)
-  if data then
+  if data and not opts.raw then
```

Comment:
I think this approach is good for now, raw is only used in renewal so it's not in hot path. 
Otherwise we will need to "parse_cdata_to_pem" here.
Let's add a comment describing why opts.raw skips cache here.

Reply:
How about we separate loading from storage from parsing PEM & caching? I refactored this (see new commit).